CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## [Unreleased]
 
+## [1.23.2] - 2024-06-13
+
+> Please categorize the following changes:
```

Comment:
Is it supposed to still be here ?

Reply:
Ah, nice catch, I'll remove it.

Reply:
Thanks, fixed in https://github.com/DataDog/dd-trace-rb/pull/3714/commits/6b2ea589b43d48e7a8a80c51e50bf0b801e9c27f